plugins/it.xsemantics.dsl/src/it/xsemantics/dsl/XsemanticsModelGenerator.xtend

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
```

Comment:
* This was introduced because the `JvmModelInferrer` causes trailing white spaces.
* Note that this change here is analog to the syntax used in .mwe2 workflow files (See here: https://github.com/meysholdt/Xpect/blob/master/org.xpect.releng/xtext-examples/org.eclipse.xtext.example.arithmetics/src/org/eclipse/xtext/example/arithmetics/GenerateArithmetics.mwe2#L36).
* Surely we could change it back if you prefer that. In case we change it back, the syntax of the generated copyright headers are then of course different to yours: The first and last lines will then only have two / one asterisk symbol instead of a whole asterisk line.